src/test/kotlin/ecommerce/dto/ProductDTOTest.kt

```diff
@@ -0,0 +1,88 @@
+package ecommerce.dto
+
+import ecommerce.dto.products.OptionDTO
+import ecommerce.dto.products.ProductDTO
+import jakarta.validation.Validation
+import jakarta.validation.Validator
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import org.springframework.boot.test.context.SpringBootTest
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class ProductDTOTest {
+    private lateinit var validator: Validator
+
+    @BeforeEach
+    fun setUp() {
+        validator = Validation.buildDefaultValidatorFactory().validator
+    }
```

Comment:
```suggestion
class ProductDTOTest {
    private lateinit var validator: Validator

    @BeforeEach
    fun setUp() {
        validator = Validation.buildDefaultValidatorFactory().validator
    }
```

I think the annotation might not be necessary in this case 👀